rules/config

```diff
@@ -150,7 +150,7 @@ INCLUDE_NAT = y
 INCLUDE_DHCP_RELAY = y
 
 # INCLUDE_DHCP_SERVER - build and install dhcp-server package
-INCLUDE_DHCP_SERVER ?= n
+INCLUDE_DHCP_SERVER = y
```

Comment:
Why DHCP server package change? Any additional checks?

Reply:
DHCP server is needed for DPU ip address assignment

Reply:
Yeah, but this change will enable DHCP server for all the platforms. Could you add Smartswitch checks and enable DHCP server?

Reply:
Done

Reply:
@kperumalbfn